src/treemap.ts

```diff
@@ -101,6 +101,7 @@ export class TreeMap {
     if (node.dom) return node.dom;
     const dom = document.createElement('div');
     dom.className = NODE_CSS_CLASS;
+    dom.tabIndex = -1;
```

Comment:
I think the currently selected element (initially the root) needs a tabIndex of 0 since otherwise tabbing from the browser chrome into the page won't work, you have to put focus in manually with the mouse.

We do this a lot in the DevTools frontend by swapping the tabIndex of the selected element with 0 and then back to -1 when it changes to another element.

Reply:
tabindex of 0 on the root applied.
also apply tabindex of 0 to currently focused item (and the root one remains).

feels pretty good, though I imagine it could get fancier.